client/src/App.js

```diff
@@ -39,12 +40,13 @@ function App() {
             <Route path="/" element={<Landing />} />
             <Route path="/login" element={<Login />} />
             <Route path="/signup" element={<Registration />} />
+            <Route path="/reset" element={<PasswordReset />} />
             <Route path="/listing/new" element={<New />} />
             <Route path="/listing/:id" element={<SingleListing />} />
             <Route path="/admin//*" element={<Admin />} />
             <Route path="/properties" element={<Properties />} />
             <Route path="/contacts" element={<Contacts />} />
-            <Route path="/about" element={<About/>}/>
+            <Route path="/about" element={<About />} />
```

Comment:
Formatting changes should really be checked in the diff when you create a PR.  If the formatting is changed the person submitting new code should ensure that the only changes picked up here in GitHub are actual code changes.  This can become a much bigger issue when collaborating with a larger team and larger codebase.

Reply:
will keep in mind. however, here im just making About look just like the other routes